accounting/bank/setup/bank_accounts.rst

```diff
@@ -0,0 +1,95 @@
+=============
+Bank Accounts
+=============
+
+You can manage as many **Bank Accounts** as needed on your database. Configuring them well allows
+you to make sure that all your banking data is up to date and ready for the reconciliation with your
+Journal Entries.
+
+In Odoo Accounting, each Bank Account is configured to have a dedicated *Journal* which is
+configured to post all entries in a dedicated *Account*.
+
+.. note:: Whenever you add a Bank Account, a dedicated journal and a dedicated account are
+   automatically created and configured.
+
+Every **Bank Journal** is displayed by default on the **Accounting Overview** in the form of a
+convenient card. It includes action buttons that are displayed when appropriate.
+
+.. image:: media/bank_accounts_card.png
+   :align: center
+   :alt: Bank Journals Cards are displayed on the Accounting Overview in Odoo Accounting
+
+.. _bank_accounts_add:
```

Comment:
```suggestion
.. _bank_accounts_add:
```

What is it for?

Reply:
It is for internal links, even from other docs.
You can use this tag to go directly to this part.
here is the explanation :)
https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#internal-hyperlink-targets

Reply:
Ok but when building your doc, it doesn't appear... :thinking:

Reply:
It's not meant to appear anywhere`, visually.
As described in the rst cheat sheet, it's a way to easily jump to that doc, at that place.
So I could, in a completely different doc, write:
```Click :ref:`here <bank_accounts_add>` for more information.```
and the link would bring the user to that doc, at that title, rather than at the top of the page.